core/src/main/scala/sttp/client4/internal/ToCurlConverter.scala

```diff
@@ -14,10 +14,21 @@ private[client4] object ToCurlConverter {
   def apply(request: GenericRequest[_, _], sensitiveHeaders: Set[String]): String =
     apply(request, sensitiveHeaders, omitAcceptEncoding = false)
 
-  def apply(request: GenericRequest[_, _], sensitiveHeaders: Set[String], omitAcceptEncoding: Boolean): String = {
+  def apply(request: GenericRequest[_, _], sensitiveHeaders: Set[String], sensitiveQueryParams: Set[String]): String =
+    apply(request, sensitiveHeaders, omitAcceptEncoding = false, sensitiveQueryParams)
+
+  def apply(request: GenericRequest[_, _], sensitiveHeaders: Set[String], omitAcceptEncoding: Boolean): String =
+    apply(request, sensitiveHeaders, omitAcceptEncoding, Set.empty)
+
+  def apply(
```

Comment:
maybe we should simplify here and just keep a single constructor with all the parameters?